### DISCUSSION TITLE
Customizing playback rate in settings menu

### DISCUSSION THREAD

**Discussion by solhosty** [muxinc/media-chrome#928] [Q&A] [answered]
I am trying to customize the playback rate in the settings menu to show my custom list, but it appears to only show these. 

Is there any known way to do this? I've tried the methods on the documentation but they do not pertain to the settings menu rates exactly. 

Here are the defaults that show: 

![image](https://github.com/muxinc/media-chrome/assets/104146303/0cc67752-a439-4cc9-b6e6-2f0863984d21)

Thanks!

**Comment by dylanjha** [comment, marked as answer]
@solhosty thanks for the question. Can you share your code? Or try to create a repro example?

Seems to be working for me here:

https://codesandbox.io/p/sandbox/funny-mestorf-69n2zs?file=/index.html:9,23

![Screenshot 2024-05-29 at 3 10 05 PM](https://github.com/muxinc/media-chrome/assets/764988/b590afd1-83ba-41bf-ace7-9bdf34e027bd)

**Reply by FairMARYAM** [reply]
#mam
